framework/build.gradle

```diff
@@ -2,12 +2,14 @@ plugins {
     id "org.gradle.test-retry" version "1.5.9"
     id "org.sonarqube" version "2.6"
     id "com.gorylenko.gradle-git-properties" version "2.4.1"
+    id "io.github.surpsg.offlins" version "0.3.0"
```

Comment:
What is the purpose of introducing this dependency?

Reply:
To solve the incompatibility problem between jacoco and powermock

Reply:
To solve the incompatibility problem between jacoco and powermock